### ISSUE TITLE
GDAL `No such file or directory` when using unzipped SAFE file

### ISSUE THREAD

**Issue by forrestfwilliams** [opera-adt/RTC#80]
Currently, `rtc_s1.py` fails with a GDAL `No such file or directory` error if the input Sentinel-1 data is unzipped.

This can be recreated by processing the Los Angeles sample data hosted on [zenodo](https://zenodo.org/record/7753472/files/s1b_los_angeles.tar.gz?download=1), unzipping the provide SAFE file, then running `rtc_s1.py` with [this configuration file](https://github.com/opera-adt/RTC/files/13743948/error.txt).

This error occurs because calling s1reader's `Sentinel1BurstSlc.slc_to_vrt_file()` method writes a VRT representing the burst SLC data with a relative path to the parent geotiff **that assumes the VRT is in the same directory as the SAFE file**. However, in RTC this VRT is written to a temporary subdirectory with this structure:

```
input_dir
└── S1_SLC.SAFE
scratch_dir
└── temp_*
   └── temp_*
      └── BURST_ID
         └── slc_VV.vrt
```

Thus, GDAL is not able to locate the parent geotiff using the information in the VRT.

I can think of two ways to remedy this:
1. Modify `Sentinel1BurstSlc.slc_to_vrt_file()` so that it uses absolute file paths
2. Force RTC to write burst VRTs into the same directory as the input SAFE file

Personally, I prefer option 1 because it fixes the problem for other projects as well, and requires fewer code changes.
[error.txt](https://github.com/opera-adt/RTC/files/13743948/error.txt)

**Comment by scottstanie** [comment]
My vote was also for (1) when I opened that issue here https://github.com/isce-framework/s1-reader/issues/96

**Comment by forrestfwilliams** [comment]
Yep, I'd prefer option 1 as well. Thanks for pointing me towards the s1-reader issue!

**Comment by gshiroma** [comment]
Thank you, @forrestfwilliams , for reporting this issue. I agree with you and @scottstanie , that would be better to fix s1-reader. Thank you, @scottstanie , for creating the issue https://github.com/isce-framework/s1-reader/issues/96 in that repository.
